.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
I think we should use/build-upon the new devcontainer.json here:

https://github.com/Azure/avm-terraform-governance/blob/main/managed-files/root/.devcontainer/devcontainer.json

Happy to copy it into this repo, but this file uses the new image and makes sure terraform is installed so all the extensions can run